### PR TITLE
chore(Federated Identity Management System): [IAFIMS-6] Implements FIMS webview component and screens

### PR DESCRIPTION
**Pull request by CrisTofani**
## Short description
This PR implements the screen to handle SSO on FIMS system through a webview

## How to test
1. Enables FIMS local FF
2. execute from anywhere:
```typescript
        handleInternalLink(
          dispatch,
          `ioit://${FIMS_ROUTES.WEBVIEW}?url=http://127.0.0.1:3000/myportal_playground.html`
        );
```
check the cookie is correctly passed out.

### CODE REVIEW

File: ts/features/fims/screens/FimsWebviewScreen.tsx
Comment (francescopersico) [inline ts/features/fims/screens/FimsWebviewScreen.tsx:71]: The cookie must be set for the domain of the FIMS provider.
I suggest adding a env var like FIMS_DOMAIN and then use it here.

**Review by francescopersico — CHANGES_REQUESTED** [review]

File: ts/features/fims/screens/FimsWebviewScreen.tsx
Comment (CrisTofani) [reply ts/features/fims/screens/FimsWebviewScreen.tsx:71]: Yes I was actually thinking about it, instead of a locally defined ENV var I would suggest to add a new information on the specific section of remote configuration, maybe we can face it in a dedicated task

Comment (CrisTofani) [reply ts/features/fims/screens/FimsWebviewScreen.tsx:71]: @francescopersico [here](https://pagopa.atlassian.net/browse/IAFIMS-7) the specific task

Comment (francescopersico) [reply ts/features/fims/screens/FimsWebviewScreen.tsx:71]: Very good!

**Review by dgopsq — CHANGES_REQUESTED** [review]
It's working fine for me 🙂 I just added two more comments but nothing really big. Since this PR is going to add small additions in the navigation, should we wait for the new **v5** before the merge or we can proceed anyway?

File: ts/features/fims/screens/FimsWebviewScreen.tsx
Comment (CrisTofani) [reply ts/features/fims/screens/FimsWebviewScreen.tsx]: Nice idea, [3a9ec41](https://github.com/pagopa/io-app/pull/3850/commits/3a9ec41ed1d8fa24fbe33faf37a7cb36e1d6950b) includes the first 2 checks in a memoized variable, the `maybeParams.isRight()` is still directly on the component to avoid compilation error

File: ts/features/fims/screens/FimsWebviewScreen.tsx
Comment (CrisTofani) [reply ts/features/fims/screens/FimsWebviewScreen.tsx]: [c9b449d](https://github.com/pagopa/io-app/pull/3850/commits/c9b449d9c064eee5ffb1b0871b662ac7903e0bcf) All cookie's utilities are now under a common file

**Review by dgopsq — APPROVED** [review]
LGTM ☄️

**Pull request merged**
